draft-ietf-acme-acme.md

```diff
@@ -1765,15 +1762,14 @@ Revocation requests are different from other ACME request in that they can be
 signed either with an account key pair or the key pair in the certificate.
 Before revoking a certificate, the server MUST verify that the key used to sign
 the request is authorized to revoke the certificate.  The server SHOULD consider
-at least the following accounts authorized for a given certificate:
+at least the following keys authorized to revoke a given certificate:
```

Comment:
Please leave this section as-is. We recently went through one round of revisions to bring it to the state it's in today. In particular, I want to be clear that accounts are authorized as opposed to keys, with the one exception of the revoke-by-private-key.